pages/error.tsx

```diff
@@ -1,18 +1,17 @@
 import { NextPage } from "next";
-import { useRouter } from "next/router";
+import {
+  BannerContent,
+  BannerImage,
+  BannerLayout,
+} from "../components/ResultBanner";
 
-const Home: NextPage = () => {
-  const { query } = useRouter();
+const Error: NextPage = () => {
   return (
-    <>
-      <h1>
-        Uh oh! We weren&apos;t able to verify you, double check that you arent
-        already registered and contact an administrator and include the code
-        below if the problem persists after trying the verification link again
-      </h1>
-      <pre>{query.code}</pre>
-    </>
+    <BannerLayout>
+      <BannerImage source="/Sad Frank.svg"/>
+      <BannerContent success={false} header="Uh oh! Register Failed" />
```

Comment:
"Register Failed" -> "Registration Failed"

Reply:
also can you change the text to "Contact an administrator by clicking the link below and asking in #register-help"

Reply:
done